docs/source/ssl.md

```diff
@@ -0,0 +1,15 @@
+# Internal SSL
+
+JupyterHub 1.0 introduces internal_ssl configuration for encryption and authentication of all internal communication.
+
+Kubespawner can mount the internal_ssl certificates as Kubernetes secrets into the jupyter user's pod.
+
+## Setup
+
+```
+c.JupyterHub.internal_ssl = True
+
+c.JupyterHub.spawner_class = 'kubespawner.KubeSpawner'
+c.KubeSpawner.internal_ssl_directory = '/srv/jupyterhub/internal-ssl/'
+c.KubeSpawner.working_dir = '/home/jovyan/' # This is required to allow us to mount the certificates into the correct directory
```

Comment:
I don't understand this section, the certificates are not mounted there anyhow right? They are mounted in /etc/jupyterhub/ssl as I understand it.

If this is needed, perhaps we could suggest the use of the fact that worker_dir will be expanded. I'm not sure if any of these would make sense though, because perhaps the username refers to the JH username while we want to work from $HOME wherever that is, which can be different from the JH username.

```
        return template.format(
            userid=self.user.id,
            username=safe_username,
            unescaped_username=self.user.name,
            legacy_escape_username=legacy_escaped_username,
            servername=safe_servername,
            unescaped_servername=servername
        )
```

Reply:
You are correct `working_dir` is no longer needed.